src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -572,27 +572,15 @@ pal::string_t resolve_sdk_version(pal::string_t sdk_path)
 
     pal::readdir(sdk_path, &versions);
     fx_ver_t max_ver(-1, -1, -1);
-    fx_ver_t max_pre(-1, -1, -1);
     for (const auto& version : versions)
     {
         trace::verbose(_X("Considering version... [%s]"), version.c_str());
 
         fx_ver_t ver(-1, -1, -1);
-        if (fx_ver_t::parse(version, &ver, true))
+        if (fx_ver_t::parse(version, &ver, false))  // false -- implies both production and prerelease.
```

Comment:
This change looks fine. While you are at this, you may want to fix https://github.com/dotnet/core-setup/issues/1760 as well since the fix is going to be right here.